portafly/src/tests/examples/accounts.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
These kind of objects are gonna be built with factories, covered in https://github.com/3scale/porta/pull/1806

Reply:
Moving discussion to https://github.com/3scale/porta/pull/1785#discussion_r402928169